sign.ps1

```diff
@@ -91,7 +91,7 @@ if ($dc_webhook) {
 
 foreach ($cookie in $conf.cookies) {
 	# Basic check if cookie valid
-	if (-not(($cookie -match 'ltoken(?:_v2)?=(?:v2_)?[0-9a-zA-Z]{40,128}') -and ($cookie -match 'ltuid(?:_v2)?=(\d+)'))) {
+	if (-not(($cookie -match 'ltoken=[0-9a-zA-Z]{40}') -and ($cookie -match 'ltuid=(\d+)')) -and -not(($cookie -match 'ltoken_v2=v2_[a-zA-Z0-9]{114}') -and ($cookie -match 'ltuid_v2=(\d+)') -and ($cookie -match 'ltmid_v2=[0-9a-zA-Z_]{13}'))) {

```

Comment:
```suggestion
	if (-not(($cookie -match 'ltoken(?:_v2)?=(?:v2_)?[0-9a-zA-Z]{40,114}') -and ($cookie -match 'ltuid(?:_v2)?=(\d+)'))) {
```
Use original RegEx but exact number of letters
move ltmid check forward or check if it exists at first when v2 only